packages/x-provider/src/signable-actions/withdrawal-actions/completeERC20Withdrawal.ts

```diff
@@ -17,6 +23,80 @@ type CompleteERC20WithdrawalWorkflowParams = {
 
 const ERC20TokenType = 'ERC20';
 
+export async function executeRegisterAndWithdrawAllERC20(
```

Comment:
Given this method will be used to withdraw ETH as well, we can rename ERC20 to Fungible instead. Or at least add a comment in all functions related to it explaining that this can also be used for ETH withdrawals.